x-pack/plugins/observability_solution/observability_ai_assistant_app/public/functions/visualize_esql.tsx

```diff
@@ -320,6 +351,7 @@ export function registerVisualizeQueryRenderFunction({
       const typedResponse = response as VisualizeQueryResponse;
 
       const columns = 'data' in typedResponse ? typedResponse.data.columns : typedResponse.content;
+      const errorMessages = 'data' in typedResponse ? typedResponse.data.errorMessages : [];
```

Comment:
should we use the one from `content`?

Reply:
done 05398aceb2965f205104dd2dc30783b8c258702a